### ISSUE TITLE
Src folder not being created when creating new project (Unpackaged folder) on Windows 7

### ISSUE THREAD

**Issue by rajesh-k** [joeferraro/MavensMate-Atom#126]
The projects created in Eclipse Force.com IDE have src folder for storing meta-data elements. The MavensMate app uses "unpackaged" folder instead of src. My question is, can MavensMate, project creation screen/page provide an option to choose src or unpackage folder for storing the meta-data?

**Comment by joeferraro** [comment]
It should not use `unpackaged`, so there must be a bug. Can you include additional information about your install? Platform? Atom version? Package version? etc.

**Comment by rajesh-k** [comment]
I am using 
- Mavens Mate Atom 0.0.14 on Windows 7 platform.
- Atom version 0.165.0

I created a project against Spring '15 pre-release.

**Comment by joeferraro** [comment]
When you create a project, is there any debug information in the Atom console?

**Comment by rajesh-k** [comment]
I didn't have Atom console open when the project with unpackaged folder was created.

I am creating another project against the same pre-release org, the project creation is still in progress and the console already showing an error  (see the screenshot)
![mavensmate-atom-issue_126](https://cloud.githubusercontent.com/assets/3512420/5601141/7b34f140-92b4-11e4-84af-2b847bb59d6e.PNG)

I will update the issue with more details when the project creation step completes.

**Comment by rajesh-k** [comment]
The previous project creation attempt didn't succeed. I clicked on the window with the red text as shown in the previous screenshot and that closed the "Creating new ...project" window.

I tried to create another project against the same org and this time I am getting a different error (see the screenshot). I am not sure what I am doing wrong.

![image](https://cloud.githubusercontent.com/assets/3512420/5601162/bfe0d948-92b5-11e4-8abd-e2f4d0708086.png)
